### DISCUSSION TITLE
homepage bug (beta for you tab)

### DISCUSSION THREAD

**Discussion by frankie303** [community/community#55141] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hello team, I encountered a bug in the homepage (when the _Beta For you_ tab is opened particularly). You can see it in the image below:

<img width="623" alt="Screenshot 2023-05-11 at 10 11 02" src="https://github.com/community/community/assets/45823795/799d2a50-cd4c-404d-b008-6def8867fdbb">

Thanks!

**Comment by honguyenminh** [comment]
Same happened on latest stable firefox

**Comment by jchv** [comment]
This seems to happen for me when a particularly long "For You" entry occurs, forcing the main column to be too wide.
![Screenshot of a GitHub "For You" entry showing a coalesced event of two repositories being created by asiekierka.](https://github.com/community/community/assets/938744/4f695223-3e27-4923-8986-808dbe1ffec9)
It probably does not matter, but I am in LibreWolf 113.0-1, makes no difference if resistFingerprinting is on or not.
